assets/js/blocks/cart-checkout/cart/full-cart/index.js

```diff
@@ -26,28 +30,62 @@ const Cart = () => {
 		// eslint-disable-next-line no-console
 		console.log( 'coupon activated: ' + couponCode );
 	};
+	const cartTotals = {
+		currency: 'EUR',
+		currency_minor_unit: 2,
+		total_items: 6000,
+		total_items_tax: 0,
+		total_fees: 0,
+		total_fees_tax: 0,
+		total_discount: 0,
+		total_discount_tax: 0,
+		total_shipping: 0,
+		total_shipping_tax: 0,
+		total_tax: 0,
+		total_price: 6000,
```

Comment:
All these values coming from the rest request are of string type. So our sample data should be string too. We'll have to watch for the potential gotcha with `'0'` because it doesn't coerce to the same boolean value as `0`.

Reply:
My bad, I changed it in the last moment thinking that was a bug in the API. :man_facepalming: 

Should be fixed in e687eafcbf6005aafee264e5b6c826c6093f367b. I moved the code that generates the table rows to a function to make it easier to read.